nixos/modules/system/boot/networkd.nix

```diff
@@ -253,22 +464,35 @@ let
 
       sectionFooOverUDP = checkUnitConfig "FooOverUDP" [
         (assertOnlyFields [
-          "Port"
           "Encapsulation"
+          "Port"
+          "PeerPort"
           "Protocol"
+          "Peer"
+          "Local"
```

Comment:
just fyi: these are missing because there's no way (that we've found) to make them work properly in the nixos framework. they only work if the addresses are available when the fou netdev is created, which never happens with dhcp and is really hard with static addressing due to how nixos orders its units.

Reply:
```suggestion
```